examples/js/loaders/TDSLoader.js

```diff
@@ -0,0 +1,830 @@
+/*
+ * Autodesk 3DS threee.js file loader, based on lib3ds.
+ * 
+ * Loads geometry with uv and materials basic properties.
+ * 
+ * @author @tentone
+ * @author @timknip
+ */
+
+"use strict";
+
+THREE.TDSLoader = function(manager)
+{

```

Comment:
@mrdoob https://gist.github.com/WestLangley/604efc8fed3d9c5edfa83d4d184c6d90 Hopefully no mistakes. mrdoobapproves had a difficult time with it. :)